src/settings.js

```diff
@@ -480,7 +480,7 @@ var EMTERPRETIFY_ADVISE = 0; // Performs a static analysis to suggest which func
                              // all things reachable from that function, as they may need to be in the YIELDLIST as well).
                              // Note that this depends on things like inlining. If you run this with different inlining than
                              // when you use the list, it might not work.
-
+var IDBFS_DEFER_WRITE = 0; // Set a timer to write into IndexedDB when file changes.
```

Comment:
how about `IDBFS_AUTOMATIC_SYNC`, since this automatic syncing also happens on exit?